src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo Moyo"
+        />
+      </a>
+
+      <nav class="menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+              data-qa="hover"
```

Comment:
Attributes of the `a` element should start on a new line with 2-space indentation when there are more than 2 attributes or long attribute values.